mlflow/tracing/utils/otlp.py

```diff
@@ -60,3 +62,68 @@ def _get_otlp_protocol() -> str:
     return os.environ.get("OTEL_EXPORTER_OTLP_TRACES_PROTOCOL") or os.environ.get(
         "OTEL_EXPORTER_OTLP_PROTOCOL", "grpc"
     )
+
+
+def _otel_proto_bytes_to_id(id_bytes: bytes) -> int:
+    """Convert OTel protobuf bytes to integer ID."""
+    return int.from_bytes(id_bytes, byteorder="big", signed=False)
+
+
+def set_otel_proto_anyvalue(pb_any_value: AnyValue, value: Any) -> None:
```

Comment:
```suggestion
def _set_otel_proto_anyvalue(pb_any_value: AnyValue, value: Any) -> None:
```

can we make this private?

1. `dev/check_function_signatures.py` thinks this is a public function and warns you when a new required parameter is added.
2. If this function is imported in a documented module, this would also be documented.

Reply:
Absolutely! Done!